cxx/util_io.cc

```diff
@@ -46,56 +74,27 @@ T_schema load_schema(const std::string& path) {
       continue;
     }
 
-    if (tokens.size() < 6) {
-      printf("Error parsing schema line %s:  expected at least six tokens, "
-             "got %ld instead.\n", line.c_str(), tokens.size());
-      assert(false);
-    }
+    size_t i = 0;
+    TYPE_CHECK(i, identifier);
```

Comment:
Out of curiosity, where is `identifier` defined? I don't see it.

Reply:
See above explanation.